### DISCUSSION TITLE
How to map_to_cell for a tetrahedral element?

### DISCUSSION THREAD

**Discussion by prusso630** [mscroggs/symfem#243] [Q&A] [answered]
```
import symfem
from symfem.symbols import x

vs = [(0,0,1), (0, -1, 0), (-1, 0,0), (0, 1,0)]
element = symfem.create_element("tetrahedron", "Lagrange", 1)
ref = symfem.create_reference("tetrahedron", vertices=vs) 
### element.map_to_cell(vs,element.get_basis_functions(),)
print(element.get_basis_functions()) 
print('done')
```

I didn't quite get it yet what map_to_cell should look like for a tetrahedron. Could an example be provided for how to map_to_cell for tet?

**Comment by mscroggs** [comment, marked as answer]
`element.map_to_cell(vs,element.get_basis_functions())` is the correct usage. This will return the basis functions mapped to the cell definite by the vertices `vs`. Try doing `print(element.map_to_cell(vs,element.get_basis_functions()))`.

If you want an explanation of what mathematical operation the map does, it's written down [here](https://defelement.com/ciarlet.html#Mapping+finite+elements).
